libs/langchain-cloudflare/package.json

```diff
@@ -38,12 +38,11 @@
   "author": "LangChain",
   "license": "MIT",
   "dependencies": {
-    "@cloudflare/ai": "^1.0.47",
     "@langchain/core": "~0.1",
     "uuid": "^9.0.1"
   },
   "devDependencies": {
-    "@cloudflare/workers-types": "^4.20231218.0",
+    "@cloudflare/workers-types": "^4.20240502.0",
```

Comment:
If we're replacing `@cloudflare/ai` with this, it needs to be moved out of dev deps and into standard deps

Reply:
`workers-types` only provides the types for the `Ai` - there's nothing runtime that would need this to be a dependency and not a devDependency`. The runtime for the AI binding is handled entirely in workerd via wrangler.